client/my-sites/upgrades/domain-management/dns/a-record.jsx

```diff
@@ -40,7 +40,7 @@ const ARecord = React.createClass( {
 			<div className={ classes }>
 				<FormFieldset>
 					<FormLabel>{ this.translate( 'Name', { context: 'Dns Record' } ) }</FormLabel>
-					{ ! isValid( 'name' ) ? <FormInputValidation text={ this.translate( 'Invalid Name' ) } isError="true" /> : null }
+					{ ! isValid( 'name' ) ? <FormInputValidation text={ this.translate( 'Invalid Name' ) } isError={ true } /> : null }
```

Comment:
A quick note that you can omit the value of `isError` since [it will be interpreted as `true`](https://facebook.github.io/react/docs/jsx-in-depth.html#boolean-attributes).